templates/yo_package.go.tpl

```diff
@@ -5,7 +5,9 @@ package {{ .Package }}
 import (
 	"context"
 	"fmt"
+	"time"
```

Comment:
why are these packages required?

Reply:
Oh.. no need this package now. I will delete it

Reply:
@kazegusuri Fixed it, please check again later